examples/nemo/eg6/README.md

```diff
@@ -0,0 +1,98 @@
+# PSyclone NEMO Example 6 - Read-only Verification
```

Comment:
Please add a brief section to examples/nemo/README.md to mention this new example.

Reply:
So many places to modify :)  Done, thanks for spotting that.